docs/meps/mep-0002.md

```diff
@@ -324,7 +324,8 @@ Files that are outside of the table of contents of the project and are reference
 The links should follow the [link AST](https://www.myst.tools/docs/spec/myst-schema#link) for external links.
 For internal project cross-references, these should be resolved to a `crossReference` node ([spec](https://www.myst.tools/docs/spec/myst-schema#crossreference)).
 
-For external project links, these extend the link object with additional data that includes the url source (`urlSource`), the scheme name (e.g. `project` or `download`), whether the link is internal (e.g. `false`), and additional optional metadata about the page that may be helpful to a renderer.
+For external project links, we will extend the link object with additional data that includes the url source (`urlSource`), the scheme name (e.g. `project` or `download`), whether the link is internal (e.g. `false`), and additional optional metadata about the page that may be helpful to a renderer.
+A prototype of this extended link type is currently defined in [myst-transforms](https://github.com/executablebooks/mystjs/blob/327c19c75c2f743e9f0736586b5232a45e00123e/packages/myst-transforms/src/links/types.ts#L4-L10).
```

Comment:
I would rather this went in the reference section, since it is mentioning an implementation and not a spec. Bit it’s not the end of the world

Reply:
It’s probably better to just copy the actual « spec proposal » here

Reply:
Yeah, agreed. Moved to the references!